python/lsst/meas/modelfit/mixture.cc

```diff
@@ -110,7 +110,11 @@ static PyMixture declareMixture(py::module &mod) {
                                 Mixture::evaluate,
             "x"_a, "p"_a);
     cls.def("evaluateComponents", &Mixture::evaluateComponents, "x"_a, "p"_a);
-    cls.def("evaluateDerivatives", &Mixture::evaluateDerivatives, "x"_a, "gradient"_a, "hessian"_a);
+    cls.def("evaluateDerivatives",
+            (void (Mixture::*)(ndarray::Array<Scalar const, 1, 1> const &,
+                               ndarray::Array<Scalar,1,1> const &,
+                               ndarray::Array<Scalar,2,1> const &) const)
+            &Mixture::evaluateDerivatives, "x"_a, "gradient"_a, "hessian"_a);
```

Comment:
Should use [`py::overload_cast`](https://pybind11.readthedocs.io/en/stable/classes.html#overloaded-methods) instead.